packages/client/src/components/SafeSettings.js

```diff
@@ -186,16 +186,15 @@ const EditSignatureThreshold = ({
   );
 };
 
-const EditSafeOwner = ({ web3, safeOwner, onCancel, onSubmit }) => {
+const RemoveSafeOwner = ({ web3, safeOwner, onCancel, onSubmit }) => {
   const { isAddressValid } = useAddressValidation(web3.injectedProvider);
   const [name, setName] = useState(safeOwner.name);
   const [address, setAddress] = useState(safeOwner.address);
   const [addressValid, setAddressValid] = useState(true);
   const isFormValid = useMemo(() => {
     return (
       name.trim().length > 0 &&
-      addressValid &&
-      (name !== safeOwner.name || address !== safeOwner.address)
+      addressValid
     );
   }, [safeOwner, name, address, addressValid]);
```

Comment:
React warning: React Hook useMemo has unnecessary dependencies: 'address' and 'safeOwner'. Either exclude them or remove the dependency array

Reply:
fixed, removed the useMemo for reducing unnecessary optimization, here's a blog post explains it well if anyone's intrested: https://kentcdodds.com/blog/usememo-and-usecallback